_content/tour/fre/functions.article

```diff
@@ -110,73 +111,72 @@ arguments to a function. To define a variadic parameter, you use an ellipsis
        return sum
    }
 
-You can call this function with any number of integers.
+Vous pouvez appeler cette fonction avec n'importe quel nombre d'entiers.
 
-** Anonymous Functions
+** Fonctions Anonymes
 
-Go supports anonymous functions, also known as closures. These are functions
-without a name, and they can be assigned to variables and used as arguments to
-other functions. Closures are often used in Go for tasks like defining inline
-functions or for concurrent programming using goroutines.
+Go prend en charge les fonctions anonymes, également appelées closures. Il s'agit de fonctions sans nom, 
+qui peuvent être assignées à des variables et utilisées comme arguments par d'autres fonctions. 
+Les closures sont souvent utilisées en Go pour des tâches telles que la définition de fonctions en ligne 
+ou la programmation concurrente à l'aide de goroutines.
 
    add := func(x, y int) int {
        return x + y
    }
 
    result := add(3, 5)
 
-** Function as a Type
+** Fonction en tant que Type
 
-In Go, functions can also be used as types. This allows you to define functions
-that take other functions as arguments or return them as results. It's a powerful
-feature for implementing higher-order functions and callbacks.
+En Go, les fonctions peuvent également être utilisées comme des types. Cela permet de définir des fonctions 
+qui prennent d'autres fonctions en arguments ou qui en retournent. C'est une fonctionnalité 
+puissante pour mettre en œuvre des fonctions d'ordre supérieur et des callbacks.
```

Comment:
"fonctions d'ordre supérieur" seems strange to read

Reply:
Alors oui :), je ne savais pas vraiment comment traduire cela, j'ai pensé à "fonctions mères", "fonctions englobantes" ou "fonctions parentes". Dans le doute j'ai fait un traduction littérale dans l'espoir d'une suggestion :))